tests/foreman/api/test_capsulecontent.py

```diff
@@ -1651,6 +1665,112 @@ def test_positive_content_counts_blank_update(
             counts is None or len(counts['content_view_versions']) == 0
         ), f"No content counts expected, but got:\n{counts['content_view_versions']}."
 
+    @pytest.mark.skip_if_open('SAT-25542')
+    @pytest.mark.parametrize('module_autosync_setting', [True], indirect=True)
+    @pytest.mark.parametrize(
+        'setting_update', ['automatic_content_count_updates=False'], indirect=True
+    )
+    def test_automatic_content_counts_update_toggle(
+        self,
+        target_sat,
+        module_capsule_configured,
+        module_autosync_setting,
+        setting_update,
+        function_org,
+        function_product,
+        function_lce,
+    ):
+        """Verify the automatic content counts update can be turned off and on again.
+
+        :id: aa8d50e3-c04c-4e0f-a1c2-544767331973
+
+        :setup:
+            1. Satellite with registered external Capsule.
+            2. foreman_proxy_content_auto_sync setting is turned on.
+            3. automatic_content_count_updates setting is turned off.
+
+        :steps:
+            1. Sync some content to the Capsule, capsule is synced automatically.
+            2. Verify no content counts update task was spawned after capsule sync completed.
+            3. Invoke manual capsule sync and verify no update task was spawned again.
+            4. Turn the automatic_content_count_updates on, invoke manual capsule sync again
+               and verify the update task was spawned this time.
+
+        :expectedresults:
+            1. Capsule content counts update task respects the setting.
+
+        :CaseImportance: Medium
+
+        :BZ: 2284027
```

Comment:
@jameerpathan111 I think we should not since it's not a feature? (It's a BZ coverage only)
As you say here https://github.com/SatelliteQE/robottelo/pull/15371#discussion_r1636104958

Reply:
It's usage is not limited to feature coverage I'd say. If it's covering a BZ scenario then it counts under `Verifies` too and we shouldn't use `:BZ:` anymore anyway.  @JacobCallahan correct me if I'm wrong.